### ISSUE TITLE
Re-nicing python scripts?

### ISSUE THREAD

**Issue by CottonEaster** [Nefelim4ag/Ananicy#37]
Ananicy doesn't appear to implement the following rule.
```
# Rules for angrysearch_update_database.py 
NAME=angrysearch_update_database.py   NICE=8   IOCLASS=idle
```

Perhaps I am doing something wrong, or perhaps Ananicy can't act on particular interpreted scripts. Or should I have something like

`NAME="python angrysearch_update_database.py"   NICE=8   IOCLASS=idle`

?

(If this doesn't really count, as stands, as a bug report, then we could class it as a problem with the documentation - unless I overlook something in the documentation.)

**Comment by Nefelim4ag** [comment]
https://github.com/Nefelim4ag/Ananicy/blob/master/README.md
```
All fields except NAME are optional.
NAME used for pgrep -f -w, so you can test your rules manually.
```

**Comment by CottonEaster** [comment]
Right thanks - and I see that there is relevant material in the README that I overlooked. I am still having trouble, though, in that `pgrep` detects the relevant process (angrysearch_update_database.py) but my ananicy rule doesn't seem to nice it. A complication is that the script in question runs in muliple ways - sometimes via cron, sometimes via a parent program (angrySearch) and, for my present testing purposes, via terminal. Also, I can't find any documentation on pgrep's `-w` option.

**Comment by Nefelim4ag** [comment]
@CottonEaster , 
```
~# man pgrep
      -w, --lightweight
              Shows all thread ids instead of pids in pgrep's context.  In pkill's context this option is disabled.
```

**Comment by CottonEaster** [comment]
Thanks. It would be nice were someone to contribute some rule(s) for angrySearch.

**Comment by Nefelim4ag** [comment]
@CottonEaster ,
```
# May i did something wrong?
~ pgrep -f -w angrysearch
11859
11870
11884

```

**Comment by CottonEaster** [comment]
The thing is, I need a rule, not for the process `angrysearch` (I want that to run unniced), but for it's updater script, which is `angrysearch_update_database.py`, which is a python3 script and which can be run by either `angrysearch` or by cron or by terminal.

Testing is not easy, because once I've updated the database, updating it again completes very quickly; so I have to wait for my files to change, in order that the updater have something to do and so will not exit very quickly. Having, in the rule file,

`name=angrysearch_update_database.py`

does not seem to work and I seem to recall that 

`name=angrysearch_update_database`

did not work either, but I will try to test that latter option again. Certainly,

`pgrep -f -w angrysearch_update_database`

does seem to work, although I am not sure I have tested it under all three launch options (= (1) run via `angrysearch`, (2) run by cron, (3) run by terminal).
